### PR TITLE
CB-18176: Added use of the eventSenderService's sendEventAndNotification method in the SdxStatusService instead of just the send notification method.

### PR DESCRIPTION
**Pull request by sxxgrc**
Jira: https://jira.cloudera.com/browse/CB-18176

Objective of this was to simply use the method in the `EventSenderService` instead of what we were using before.

### CODE REVIEW

**Comment by sxxgrc** [comment]
@sodre90 can you confirm that this is what you wanted? I want to make sure to get your approval before this is merged in.

**Comment by sodre90** [comment]
Yes! Thank you @sxxgrc !

**Review by sodre90 — APPROVED** [review]
please fix spot bugs fail, anyway it looks good for me!

**Pull request merged**
